entrypoint.sh

```diff
@@ -0,0 +1,284 @@
+#!/bin/bash
+
+if [ ! -f '/opt/stalwart-mail/etc/aio-config.env' ]; then
+  mkdir -p '/opt/stalwart-mail/etc'
+  echo '## REMOVE VARIABLES BELOW IF YOU WANT TO USE DEFAULT VALUES ##' > '/opt/stalwart-mail/etc/aio-config.env'
+fi
+
+source '/opt/stalwart-mail/etc/aio-config.env'
```

Comment:
I guess here should be a check that checks if the file is there?

Reply:
85e2d06